<?xml version="1.0"?>
<launch>

     <include file="$(find swanson_sensors)/launch/multiplexed_imus.launch" />

     <node pkg="swanson_controls" type="camera_gimbal_controller_node" name="left_gimbal_node">
          <param name="imu_topic" value="imu/l/angles"/>
          <param name="target_topic" value="gimbal/left/target"/>
          <param name="actuator_channel" value="0"/>
     </node>
     <node pkg="swanson_controls" type="camera_gimbal_controller_node" name="right_gimbal_node">
          <param name="imu_topic" value="imu/r/angles"/>
          <param name="target_topic" value="gimbal/right/target"/>
          <param name="actuator_channel" value="1"/>
     </node>
     <node pkg="swanson_controls" type="camera_gimbal_controller_node" name="front_gimbal_node" output="screen">
          <param name="imu_topic" value="imu/f/angles"/>
          <param name="target_topic" value="gimbal/front/target"/>
          <param name="actuator_channel" value="2"/>
     </node>

</launch>
